File: lib-database/src/main/java/com/ustadmobile/lib/db/entities/EntryStatusResponseWithNode.java
package com.ustadmobile.lib.db.entities;

import com.ustadmobile.lib.database.annotation.UmEmbedded;

/**
 * Created by mike on 2/2/18.
 */

public class EntryStatusResponseWithNode extends EntryStatusResponse{

    @UmEmbedded
    private NetworkNode networkNode;

    public EntryStatusResponseWithNode() {

    }

    public EntryStatusResponseWithNode(NetworkNode networkNode) {
        this.networkNode = networkNode;
    }


    public NetworkNode getNetworkNode() {
        return networkNode;
    }

    public void setNetworkNode(NetworkNode networkNode) {
        this.networkNode = networkNode;
    }
}

File: lib-database-entities/src/commonMain/kotlin/com/ustadmobile/lib/db/entities/EntryStatusResponseWithNode.kt
package com.ustadmobile.lib.db.entities

import androidx.room.Embedded
import kotlinx.serialization.Serializable

/**
 * Created by mike on 2/2/18.
 */
@Serializable
class EntryStatusResponseWithNode(@Embedded var networkNode: NetworkNode? = null) : EntryStatusResponse()
